README.md

```diff
@@ -188,6 +188,141 @@ For the deidentification of the EHR extracts, we rely on an instance running the
 
+### 3. Configure a new project
+
+To configure a new project, follow these steps:
+
+1. Create a new `git` branch from `main`
+
+    ```sh
+    git checkout main
+    git pull
+    git switch -c <branch-name>
+    ```
+
+1. Copy the `template_config.yaml` file to a new file in the `projects/config` directory and fill
+   in the details.
+1. The filename of the project config should be `<project-slug>`.yaml
+
+    >[!NOTE]
+    > The project slug should match the project name in the `extract_summary.json` log file!
+
+1. [Open a PR in PIXL](https://github.com/UCLH-Foundry/PIXL/compare) to merge the new project config into `main`
+
+#### The config YAML file
+
+The configuration file defines:
+
+- Project name: the `<project-slug>` name of the Project
+- The DICOM dataset modalities to retain (e.g. `["DX", "CR"]` for X-Ray studies)
+- The anonymisation operations to be applied to the DICOM tags, by providing a file path to one or multiple YAML files
+- The endpoints used to upload the anonymised DICOM data and the public and radiology
+  [parquet files](./docs/file_types/parquet_files.md). We currently support the following endpoints:
+    - `"none"`: no upload
+    - `"ftps"`: a secure FTP server (for both _DICOM_ and _parquet_ files)
+      Requires the `FTPS_*` environment variables to be set in `.env`
+    - `"azure"`: a secure Azure Blob Storage account (for both _DICOM_ and _parquet_ files)
+      Requires the `AZURE_*` environment variables to be set in `.env`
+    - `"dicomweb"`: a DICOMweb server (for _DICOM_ files only)
+      Requires the `DICOMWEB_*` environment variables to be set in `.env`
+
+#### Project secrets {#project-secrets}
+
+Any credentials required for uploading the project's results should be stored in an **Azure Key Vault**
+(set up instructions below).
+PIXL will query this key vault for the required secrets at runtime. This requires the following
+environment variables to be set in `.secrets.env` so that PIXL can connect to the key vault:
+
+- `EXPORT_AZ_CLIENT_ID`: the service principal's client ID, mapped to `AZURE_CLIENT ID` in `docker-compose`
+- `EXPORT_AZ_CLIENT_PASSWORD`: the password, mapped to `AZURE_CLIENT_SECRET` in `docker-compose`
+- `EXPORT_AZ_TENANT_ID`: ID of the service principal's tenant. Also called its 'directory' ID. Mapped to `AZURE_TENANT_ID` in `docker-compose`
+- `EXPORT_AZ_KEY_VAULT_NAME` the name of the key vault, used to connect to the correct key vault
+
+<details><summary>Azure Keyvault setup</summary>
+
+## Azure Keyvault setup
+
+_This is done for the \_UCLH_DIF_ `dev` tenancy, will need to be done once in the _UCLHAZ_ `prod`
+tenancy when ready to deploy to production.\_
+
+This Key Vault and secret must persist any infrastructure changes so should be separate from disposable
+infrastructure services. ServicePrincipal is required to connect to the Key Vault.
+
+The application uses the ServicePrincipal and password to authenticates with Azure via environment
```

Comment:
```suggestion
The application uses the ServicePrincipal and password to authenticate with Azure via environment
```
Typo. Also, ServicePrincipal is used both as a name and as a regular noun within a couple of sentences which makes it confusing. Maybe a short explanation of what it is might help?